### ISSUE TITLE
Check and update Support Request categories to match our Quickbooks reporting

### ISSUE THREAD

**Issue by jel535s** [MidwestAccessCoalition/lockbox_rails#395]
Blocked by #349 
## Context
1. Each expense category for Support Requests should correspond with an accounting code in Quickbooks (will be addressed by #490)
2. Finance should be able to change/configure these categories (will be addressed by #469 )

## Process

- [x] Get a list of relevant Quickbooks categories from Finance. (see Resources, highlighted line items 1-12 are relevant here)

- [x] Check our Support Request forms in the Lockbox app- do our line item options match?

- [ ] If line items do not match, add or combine as needed.  (will be addressed by #490)

## Acceptance Criteria/Definition of Done
On the Support Request form, the list of categories which can be line items should match exactly the list of relevant Quickbooks categories.

## Resources
[MAC Expense Account Categories.xlsx](https://github.com/MidwestAccessCoalition/lockbox_rails/files/4335245/MAC.Expense.Account.Categories.xlsx)

**Comment by gujacq** [comment]
- Figure out how we want to configure the transaction types (to be defined in this ticket)
- Need to spec out another ticket for design interface to do so

**Comment by jel535s** [comment]
Updated and created new ticket to track engineering deliverables #490
